docs/api.md

```diff
@@ -159,6 +160,24 @@ Array values are converted into [responsive values][responsive-styles].
 <Box color='#f00' />
 ```
 
+### opacity (responsive)
```

Comment:
Thanks for the PR! I'd rather see `opacity` in the *Misc* section as I'd definitely not consider it a core part of a design system